### PR TITLE
Simplify `envtool shell` subcommands

### PR DESCRIPTION
**Pull request by AlekSi**
## Readiness checklist

* [x] I added/updated unit tests.
* [ ] I added/updated integration/compatibility tests.
* [x] I added/updated comments and checked rendering.
* [x] I made spot refactorings.
* [ ] I updated user documentation.
* [x] I ran `task all`, and it passed.
* [x] I ensured that PR title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: cmd/envtool/envtool.go [hunk 1]
```diff
@@ -331,15 +331,11 @@ func mkdir(paths []string) error {
 		}
 	}
 
-	if errs != nil {
-		return errs
-	}
-
-	return nil
```
Comment (AlekSi) [inline cmd/envtool/envtool.go:338]: We always return `errs`, after all

File: cmd/envtool/envtool.go [hunk 2]
```diff
@@ -389,22 +381,21 @@ func main() {
...
-			printDiagnosticData(err, logger)
-			os.Exit(1)
-		}
+		err = setup(ctx, logger)
 	case "shell mkdir <path>":
-		if err := mkdir(args[2:]); err != nil {
-			printDiagnosticData(err, logger)
-			os.Exit(1)
-		}
+		err = mkdir(cli.Shell.Mkdir.Paths...)
```
Comment (AlekSi) [inline cmd/envtool/envtool.go:390]: Pass already parsed arguments

File: cmd/envtool/envtool_test.go [hunk 3]
```diff
@@ -37,38 +35,26 @@ func TestPrintDiagnosticData(t *testing.T) {
...
-	err := rmdir([]string{"rz"})
+	err := rmdir("absent")
 	assert.NoError(t, err)
 }
 
 func TestMkdirAndRmdir(t *testing.T) {
 	t.Parallel()
 
-	paths := []string{"ab/c"}
+	paths := []string{"ab/c", "ab"}
```
Comment (AlekSi) [inline cmd/envtool/envtool_test.go:45]: So "ab" is not left after the test

File: cmd/envtool/envtool_test.go [hunk 4]
```diff
@@ -37,38 +35,26 @@ func TestPrintDiagnosticData(t *testing.T) {
...
 	assert.NoError(t, err)
 
-	// check if paths exist.
-	var errs error
-
 	for _, path := range paths {
-		if _, err = os.Stat(path); err != nil {
-			errs = errors.Join(errs, err)
-		}
+		assert.DirExists(t, path)
```
Comment (AlekSi) [inline cmd/envtool/envtool_test.go:51]: Useful helper

**Comment by AlekSi** [comment]
@kropidlowsky PTAL

**Comment by kropidlowsky** [comment]
@AlekSi Thanks a lot!

**Review by w84thesun — APPROVED** [review]

**Review by rumyantseva — APPROVED** [review]
Great! I tried various tasks on Windows, they work apart from `packages`.

I think the last thing we need to decide about is how to retrieve the version for `packages-deb`/`packages-rpm` because `sh: cat` doesn't work as is. Though, powershell itself supports `cat` command.

**Review by chilagrow — APPROVED** [review]

**Review by chilagrow — APPROVED** [review]

**Comment by AlekSi** [comment]
#2615

**Pull request merged**